code/game/objects/structures/flora.dm

```diff
@@ -102,6 +102,8 @@
 /obj/structure/flora/pottedplant/attackby(obj/item/W, mob/user)
 	if(!ishuman(user))
 		return
+	if(ismob(W))
+		return //no hiding mobs in there
```

Comment:
does this check for holders? since held animals are holders

Reply:
fixed